src/Dapr.Actors/Runtime/ActorHost.cs

```diff
@@ -115,20 +116,23 @@ public static ActorHost CreateForTest(Type actorType, string actorTypeName, Acto
         /// <param name="loggerFactory">The logger factory.</param>
         /// <param name="proxyFactory">The <see cref="ActorProxyFactory" />.</param>
         /// <param name="timerManager">The <see cref="ActorTimerManager" />.</param>
+        /// <param name="daprStateProvider">The <see cref="DaprStateProvider" />.</param>
         internal ActorHost(
```

Comment:
You should add a new method here with these params and keep the old one for backwards compatibility.